docs/reference/github/workflow/pr.md

```diff
@@ -0,0 +1,81 @@
+will be the issue branch.
+
+![image](../../../assets/images/github/workflow/branch_comparison.png)
+
+Once you have decided on your base and compare branches, click on `Create pull request`. You should see
+the page below:
+
+![image](../../../assets/images/github/workflow/pr_example.png)
+
+Notice how this is remarkably similar to the page of an issue. To link a pull request to an issue, simply add `#<ISSUE NUMBER>`
+to the initial comment in the pull request.
+
+!!! example
+
+    "This issue resolves #49. Please review my pull request!"
+
+Observe that the right-hand side banner contains the following:
+
+| Field               | Description                                                                                        |
+| :------------------ | :------------------------------------------------------------------------------------------------- |
+| Reviewers           | Assign reviewers to review your pull request. Always try to assign at least one reviewer.          |
+| Assignees           | Assign the people who worked on the issue.                                                         |
+| Labels              | Assign labels to categorize pull requests.                                                         |
+| Projects            | Assign a pull request to a project.                                                                |
+| Milestone           | Assign a pull request to a milestone.                                                              |
+
+If you linked the pull request to an issue, you **do not** need to add the pull request to a project or a milestone. At the bare
+minimum, you should assign the reviewers, assignees, and the labels if necessary.
+
+## Reviewing a pull request
+
+A common activity that you will participate in is reviewing pull requests, and it is an essential part of collaborating
+on projects. You will be notified when you have been requested to review a pull request and should promptly review it as
+soon as time permits.
+
+![image](../../../assets/images/github/workflow/pr_comment_snippet.png)
+
+In particular, you will most likely be doing the following in a pull request (both as the reviewer and the PR creator):
+
+- **Asking Questions:** Clarify your understanding about something that you are not sure about.
+- **Provding Suggestions:** Give some ideas about how to improve the current implementation and provide feedback to
+your peers. This is a good opportunity to share your knowledge with others.
+- **Verify Implementations:** Identify potential bugs in the implementation and raise your concerns with the person
+who developed the solution. This will reduce the likelihood of bugs and significantly bring down the number of issues
+in the future.
+- **Documentation:** Record why certain changes were made, especially if this diverges from the proposed solution
+in the linked issue (if any).
+
```

Comment:
You could add stuff about adding comments as part of a review, approve vs comment vs request changes

Reply:
My intentions for this section was to describe what you would do in a PR at a high level, not go into specifics. I did think of adding what you mentioned, but this was something that one would gain from experience with PRs (we are over documenting at this point).

Reply:
Do we even need a section for reviewing PR's then, mostly leads would be the ones reviewing, and it's pretty self explanatory

Reply:
Good point, but it could serve new members well since they have less experience and reviewing PRs should not be exclusive to leads, but should be inclusive for everyone involved.